easybuild/easyconfigs/s/SPH-EXA/SPH-EXA-develop-gompi-2021a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = "CMakeMake"
+
+name = "SPH-EXA"
+version = "develop"
```

Comment:
@jgphpc could we use a date-based or commit-based version instead of `develop`? That way, we could update to newer versions later on.